models/org_team.go

```diff
@@ -243,6 +243,21 @@ func (t *Team) addAllRepositories(e Engine) error {
 	return nil
 }
 
+// AddAllRepositories adds all repositories to the team
+func (t *Team) AddAllRepositories() (err error) {
+	sess := x.NewSession()
+	defer sess.Close()
+	if err = sess.Begin(); err != nil {
+		return err
+	}
+
+	if err = t.addAllRepositories(sess); err != nil {
+		return err
+	}
+
```

Comment:
Should there be also call to update num_repos column?

Reply:
It's maybe little bit complex/inefficient, but addAllRepositories->addRepository increments the num_repos.

Reply:
Than remove should also probably decrease automatically without having to update in that function to have consistency in code

Reply:
I moved it to the lower function (removeAllRepositories), that function do a single delete operation on all team-repositories.